modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -13,21 +13,21 @@ Before using the {product-title} (ROSA) CLI, `rosa`, to create {hcp-title-first}
 * You have completed the AWS prerequisites for {hcp-title}.
 * You have available AWS service quotas.
 * You have enabled the ROSA service in the AWS Console.
-* You have installed and configured the latest ROSA CLI (`rosa`) on your installation host.
-+
-[NOTE]
-====
-To successfully install {hcp-title} clusters, use the latest version of the ROSA CLI (`rosa`).
-====
+* You have installed and configured ROSA CLI (`rosa`) version 1.2.23 or later on your installation host.
 * You have logged in to your Red Hat account by using the ROSA CLI.
 
 .Procedure
 
-. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command: 
+. If they do not exist in your AWS account, create the required account-wide STS roles and policies by running the following command:
 +
 [source,terminal]
 ----
-$ rosa create account-roles --force-policy-creation
+$ rosa create account-roles --hosted-cp --force-policy-creation
 ----
 +
-The `--force-policy-creation` parameter updates any existing roles and policies that are present. If no roles and policies are present, the command creates these resources instead.
\ No newline at end of file
+The `--force-policy-creation` parameter updates any existing roles and policies that are present. If no roles and policies are present, the command creates these resources instead.
```

Comment:
We don't use this flag in combination with the `--hosted-cp` flag, as the policies are updated by AWS, and not created by the user or the ROSA CLI, we are only attaching existing policies.